spec/core_functions/color/adjust_color/hwb.hrx

```diff
@@ -73,7 +73,7 @@ a {b: adjust-color(#993333, $blackness: 100%)}
 
 <===> blackness/max/output.css
 a {
-  b: rgb(42.5, 42.5, 42.5);
+  b: rgb(31.875, 31.875, 31.875);
```

Comment:
Why does the `rgb()` value change by more than 10 points on each channel? I don't think the `adjust-color` function changed in a way that would cause this much of a change. Is it intended? I'm not sure what I missed 😅